src/conn/conn_api.c

```diff
@@ -1790,6 +1790,12 @@ __wt_debug_mode_config(WT_SESSION_IMPL *session, const char *cfg[])
     else
         WT_RET(__wt_calloc_def(session, conn->debug_ckpt_cnt, &conn->debug_ckpt));
 
+    WT_RET(__wt_config_gets(session, cfg, "debug_mode.cursor_copy", &cval));
+    if (cval.val)
+        F_SET(conn, WT_CONN_DEBUG_COPY_CURSOR_DATA);
+    else
+        F_CLR(conn, WT_CONN_DEBUG_COPY_CURSOR_DATA);
```

Comment:
I have a preference for this name to be `WT_CONN_DEBUG_CURSOR_COPY` to keep the words in the same order. But won't fight you if you like this naming better.

Reply:
Sure, changed the naming.